### PR TITLE
Add context path to logged endpoints

### PR DESCRIPTION
**Pull request by afrin216**
###### Problem:
Endpoints logged out take into consideration jersey's URL pattern, but not the context path.
```
GET /endpointA
POST /endpointA
```
Copy pasting this URL gives 404s if you have context-path

###### Solution:
Append context path if present to the end points along with jersey URL pattern

###### Result:
```
GET /context-path/endpointA
POST /context-path/endpointA
```

### CODE REVIEW

**Comment by nickbabcock** [comment]
Just to double check if I understand you right, now everyone that uses

```yaml
server:
  type: simple
  applicationContextPath: /application
```

Will see

```
GET /application/endpointA
```

instead of

```
GET /endpointA
```

**Comment by afrin216** [comment]
@nickbabcock Yes, that's it. So that consumers can copy pasting from the console directly get the context path without looking anywhere else (We have quite a few including myself)

**Comment by arteam** [comment]
I think this change is justified. I can imagine situations when operations staff tries to diagnose a service by the logs and expect that the logged endpoints correspondent to actual endpoints.

**Comment by arteam** [comment]
@afrin216 Thank you for the contribution!

File: dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java [hunk 1]
```diff
@@ -177,8 +185,10 @@ String cleanUpPath(String path) {
         private final String rootPath;
         private final Class<?> klass;
 
-        EndpointLogger(String urlPattern, Class<?> klass) {
-            this.rootPath = urlPattern.endsWith("/*") ? urlPattern.substring(0, urlPattern.length() - 1) : urlPattern;
+        EndpointLogger(String contextPath, String urlPattern, Class<?> klass) {
+            final String rootPattern = urlPattern.endsWith("/*") ? urlPattern.substring(0, urlPattern.length() - 1) : urlPattern;
```
Comment (Alexey1Gavrilov) [inline dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java]: Looks like by default the printed path starts with double slashes "//".
If urlPattern == '/*', then rootPattern = '/', and if contextPath == '/', rootPath = '//'

Comment (arteam) [reply dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java]: Thanks for reporting this! I've created #2279 to fix the output.

**Pull request merged**